tools/bin/acceptance_test_kube.sh

```diff
@@ -36,7 +36,6 @@ kubectl describe pods | grep "Name\|Node"
 sleep 120s
 
 if [ -n "$CI" ]; then
-  bootloader_logs () { kubectl logs pod/airbyte-bootloader > /tmp/kubernetes_logs/bootloader.txt; }
```

Comment:
we can no longer retrieve the bootloader logs since the pod is removed

Reply:
Ah, good catch!